extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -290,6 +292,9 @@ export class SubscriptionService {
         'Product id is null'
     ));
     this.platformStore_ = new PlatformStore(serviceIds);
+    this.platformConfig_['services'].forEach(service => {
+      this.initializeLocalPlatforms_(service);
```

Comment:
Why do this? This could actually be the right thing to do, but I'd like to understand your reasoning for this at this time.

Reply:
I am still keeping the entitlement in my system for any future proof use cases, pingbacks etc.
Which is right now proxied to local platform.

In order to keep an entitlement to a platform, that platform must be initialized as well.

Reply:
The concern I have which might be unfounded here is the risk of executing authorization from local platform. At the very least, can we maybe disable `getAuthorization` API in this case?